conf/styleguide.config.js

```diff
@@ -174,6 +174,14 @@ const allSections = [
                 name: 'States',
                 content: '../src/icons/states/README.md',
             },
+            {
+                name: 'Tasks',
+                components: () => [
```

Comment:
We've started to prefer a single content page for multiple small components. See `name: 'States'` above for an example.

Reply:
@jstoffan when using README.md, that requires ... `require()` for each component, correct? with the individual files, the markdown files seem to infer which component to import.

Reply:
Yes, that's true. The READMEs serve a dual purpose as Github documentation and generators for the example site. Having a list of `require` statements at the top of the README seems like a reasonable tradeoff, but if that's a concern, then the other icon READMEs will also need to be reverted to single components. 

@booc0mtaco @tjgupta, can you sync up on the desired strategy here?